test/parallel/test-crypto-hmac.js

```diff
@@ -9,11 +9,26 @@ if (!common.hasCrypto) {
 var crypto = require('crypto');
 
 // Test HMAC
-var h1 = crypto.createHmac('sha1', 'Node')
+const h1 = crypto.createHmac('sha1', 'Node')
                .update('some data')
-               .update('to hmac')
-               .digest('hex');
-assert.equal(h1, '19fd6e1ba73d9ed2224dd5094a71babe85d9a892', 'test HMAC');
+               .update('to hmac');
+assert.equal(h1.digest('hex'),
+             '19fd6e1ba73d9ed2224dd5094a71babe85d9a892',
+             'test HMAC');
+
+const h2 = crypto.createHmac('sha1', 'Node')
+               .update('some data')
+               .update('to hmac');
+assert.ok(h2.validate(
```

Comment:
Reverted, thanks. I can't check that CI URL because I'm getting 504's.

On Thu, May 12, 2016 at 5:21 PM, Fedor Indutny <notifications@github.com>
wrote:

> In test/parallel/test-crypto-hmac.js
> <https://github.com/nodejs/node/pull/3073#discussion_r63118037>:
>
> >                 .update('some data')
> > -               .update('to hmac')
> > -               .digest('hex');
> > -assert.equal(h1, '19fd6e1ba73d9ed2224dd5094a71babe85d9a892', 'test HMAC');
> > +               .update('to hmac');
> > +assert.equal(h1.digest('hex'),
> > +             '19fd6e1ba73d9ed2224dd5094a71babe85d9a892',
> > +             'test HMAC');
> > +
> > +const h2 = crypto.createHmac('sha1', 'Node')
> > +               .update('some data')
> > +               .update('to hmac');
> > +assert.ok(h2.validate(
>
> Looks like a leftover from .validate() ;)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/3073/files/8f232da78b2bbb0628a1401f9b94a9d362d12191#r63118037>
>